pandas/tests/extension/test_integer.py

```diff
@@ -249,6 +249,51 @@ class TestBooleanReduce(base.BaseBooleanReduceTests):
     pass
 
 
+class TestNumericAccumulation(base.BaseNumericAccumulateTests):
```

Comment:
we already have *some* tests for sparse, pls add the AccumulateTests mixin there as well.